variables.tf

```diff
@@ -50,14 +62,20 @@ variable "db_version" {
   default     = "MYSQL_5_7"
 }
 
-variable "db_charset" {
-  description = "The charset for the MySQL database."
+variable "default_db_name" {
+  description = "Name of the default database to be created."
+  type        = string
+  default     = "default"
```

Comment:
We could name the default db_name as mysql, to make more sense. 
Just like postgres.

Reply:
I just checked. The mysql instances already a `mysql` db for its internal stuff. So calling this db `mysql` would be conflicting. Let's keep it `default`.